digital-credentials/support/helper.js

```diff
@@ -0,0 +1,89 @@
+// @ts-check
+// Import the types from the TypeScript file
+/**
+ * @typedef {import('../dc-types').ProviderType} ProviderType
+ * @typedef {import('../dc-types').IdentityRequestProvider} IdentityRequestProvider
+ * @typedef {import('../dc-types').DigitalCredentialRequestOptions} DigitalCredentialRequestOptions
+ * @typedef {import('../dc-types').CredentialRequestOptions} CredentialRequestOptions
+ * @typedef {import('../dc-types').SendMessage} SendMessage
+ */
+/**
+ * @param {ProviderType[]} [providersToUse=["default"]] - An array that can only contain "default" or "oid4vp".
+ * @returns {CredentialRequestOptions}
+ */
+export function makeGetOptions(providersToUse = ["default"]) {
+  if (!Array.isArray(providersToUse) || !providersToUse?.length) {
+    return { digital: { providers: providersToUse } };
+  }
+  const providers = [];
+  for (const provider of providersToUse) {
+    switch (provider) {
+      case "oid4vp":
```

Comment:
```suggestion
      case "urn:openid.net:oid4vp":
```

I think this is the value that the OpenID4VP group is converging to?

https://github.com/openid/OpenID4VP/pull/155/files#diff-3118c9756a1d7b361bb53af8bd9d65666476a27cfeeced56c47a5dccc38eac55R1598

Reply:
We agreed to change it to "openid4vp". See https://github.com/openid/OpenID4VP/pull/155/files#r1660568857